lang/ar.json

```diff
@@ -30,5 +30,58 @@
   "This modal can be closed by pressing the Escape key or activating the close button.": "يمكن غلق هذه النافذة المشروطة عن طريق الضغط على زر الخروج أو تفعيل زر الإغلاق",
   ", opens captions settings dialog": ", تفتح نافذة  خيارات التعليقات",
   ", opens subtitles settings dialog": ", تفتح نافذة  خيارات الترجمة",
-  ", selected": ", مختار"
+  ", selected": ", مختار",
+  "Audio Player": "مشغل الصوت",
+  "Video Player": "مشغل الفيديو",
+  "Replay": "إعادة التشغيل",
+  "Seek to live, currently behind live": "ذهاب إلى نقطة البث المباشر، متأخر عن البث المباشر حاليًا",
+  "Seek to live, currently playing live": "ذهاب إلى نقطة البث المباشر، البث المباشر قيد التشغيل حاليًا",
+  "Progress Bar": "شريط التقدم",
+  "progress bar timing: currentTime={1} duration={2}": "توقيت شريط التقدم: currentTime={1} duration={2}",
+  "Descriptions": "الأوصاف",
+  "descriptions off": "إخفاء الأوصاف",
+  "Audio Track": "المسار الصوتي",
+  "Volume Level": "مستوى الصوت",
+  "The media is encrypted and we do not have the keys to decrypt it.": "الوسائط مشفرة وليس لدينا الرموز اللازمة لفك شفرتها.",
+  "Close Modal Dialog": "إغلاق مربع الحوار المشروط",
+  ", opens descriptions settings dialog": "، يفتح مربع حوار إعدادات الأوصاف",
+  "captions settings": "إعدادات التعليقات التوضيحية",
+  "subtitles settings": "إعدادات الترجمات",
+  "descriptions settings": "إعدادات الأوصاف",
+  "Text": "النص",
+  "White": "أبيض",
+  "Black": "أسود",
+  "Red": "أحمر",
+  "Green": "أخضر",
+  "Blue": "أزرق",
+  "Yellow": "أصفر",
+  "Magenta": "أرجواني",
+  "Cyan": "أزرق سماوي",
+  "Background": "الخلفية",
+  "Window": "نافذة",
+  "Transparent": "شفاف",
+  "Semi-Transparent": "نصف شفاف",
+  "Opaque": "معتم",
+  "Font Size": "حجم الخط",
+  "Text Edge Style": "نمط حواف النص",
+  "None": "لا شيء",
+  "Raised": "بارز",
+  "Depressed": "منخفض",
+  "Uniform": "منتظم",
+  "Dropshadow": "ظل خلفي",
+  "Font Family": "عائلة الخطوط",
+  "Proportional Sans-Serif": "Proportional Sans-Serif",
+  "Monospace Sans-Serif": "Monospace Sans-Serif",
+  "Proportional Serif": "Proportional Serif",
+  "Monospace Serif": "Monospace Serif",
+  "Casual": "Casual",
```

Comment:
Are these left untranslated because they don't relate to Arabic typography?

Reply:
That's how they came back from the translation service.